core/error.rs

```diff
@@ -192,15 +192,20 @@ impl JsError {
     let msg = v8::Exception::create_message(scope, exception);
 
     let mut exception_message = None;
-    let state_rc = JsRuntime::state(scope);
-    let state = state_rc.borrow();
-    if let Some(format_exception_cb) = &state.js_format_exception_cb {
-      let format_exception_cb = format_exception_cb.open(scope);
-      let this = v8::undefined(scope).into();
-      let formatted = format_exception_cb.call(scope, this, &[exception]);
-      if let Some(formatted) = formatted {
-        if formatted.is_string() {
-          exception_message = Some(formatted.to_rust_string_lossy(scope));
+    // Nest this state borrow. A mutable borrow can occur when accessing `stack`
```

Comment:
What do you mean by "next this state borrow"?

Reply:
`nest`, just explaining why the block scope is there.

Reply:
Ooops, typo. I meant "nest". This is fine then